src/OmniSharp.Dnx/DnxPaths.cs

```diff
@@ -138,31 +132,24 @@ private IEnumerable<string> GetRuntimeLocations()
             {
                 // Newer path
                 yield return Path.Combine(home, ".dnx");
+            }
 
-                // New path
-                yield return Path.Combine(home, ".k");
-
-                // Old path
-                yield return Path.Combine(home, ".kre");
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ProgramFiles")))
```

Comment:
I wonder who needs to update?

* dnvm - to understand the installed location?
* dnx installer - to use one of the known global locations used by `dnvm`?

The question is what does OmniSharp support on windows?

I'm leaning towards all 3 locations, because it's very possible someone use vscode would just install aspnet5 with the windows installer.

cc @glennc @DamianEdwards